### DISCUSSION TITLE
embedding model max token length

### DISCUSSION THREAD

**Discussion by hamadaelnopy** [vanna-ai/vanna#696] [Q&A]
max token for the default embedding model for Milvus is 512, some DLL exceed that easily, no way to change this

**Comment by albertusgeyser** [comment]
I got the same issue any ideas as i had to break my ddl in parts

**Reply by hamadaelnopy** [reply]
Yes you had to break your DDL if possible, for me I removed some unnecessary columns from the DDL statement
